src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export type Props = {
+  movies?: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <article className="movies">
+    {movies.map(movie => (
+      <div key={movie.imdbId}>
+        <MovieCard {...movie} />
```

Comment:
You shouldn't pass props like this. Just pass movie, and then destructure it inside the `MovieCard` component
```suggestion
        <MovieCard movie={movie} />
```